src/jaxsim/parsers/rod/utils.py

```diff
@@ -1,15 +1,23 @@
+import enum
 import os
 
 import jaxlie
 import numpy as np
 import numpy.typing as npt
 import rod
+import trimesh
 
 import jaxsim.typing as jtp
 from jaxsim.math.inertia import Inertia
 from jaxsim.parsers import descriptions
 
 
+class MeshMappingMethods(enum.Enum):
+    VertexExtraction = (0,)
+    RandomSurfaceSampling = (1,)
+    UniformSurfaceSampling = 2
```

Comment:
You can leverage `enum.IntEnum` here (see https://docs.python.org/3/library/enum.html#enum.IntEnum)
```suggestion
@enum.unique
class MeshMappingMethods(enum.IntEnum):
    VertexExtraction = enum.auto()
    RandomSurfaceSampling = enum.auto()
    UniformSurfaceSampling = enum.auto()
```